db/db_impl/db_impl_write.cc

```diff
@@ -658,7 +693,13 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
             continue;
           }
           // TODO: maybe handle the tracing status?
-          tracer_->Write(writer->batch).PermitUncheckedError();
+          if (wbwi && !ingest_wbwi_for_commit) {
+            // for transaction write, tracer only needs the commit marker which
```

Comment:
Is it intentional to not trace the wbwi->GetWriteBatch for a transaction commit write?

Reply:
Yes, the tracer only needs the commit marker which is in writer->batch. Updated comment for this.